CONTRIBUTING.md

```diff
@@ -196,62 +194,62 @@ You can set the `DEBUG` environment variable to turn on debugging logs. E.g. `DE
+To get around this, we inline some of these dependencies' types in `packages/vite/types`. This way, we can still expose the typing but bundle the dependency's source code.
 
-Use `pnpm run check-dist-types` to check bundled types does not rely on types in `devDependencies`. If you are adding `dependencies`, make sure to configure `tsconfig.check.json`.
+Use `pnpm run check-dist-types` to check that the bundled types do not rely on types in `devDependencies`. If you are adding `dependencies`, make sure to configure `tsconfig.check.json`.
 
-### Think before adding yet another option
+### Think Before Adding Yet Another Option
 
-We already have many config options, and we should avoid fixing an issue by adding yet another one. Before adding an option, try to think about:
+We already have many config options, and we should avoid fixing an issue by adding yet another one. Before adding an option, consider whether the problem:
 
-- Whether the problem is really worth addressing
-- Whether the problem can be fixed with a smarter default
-- Whether the problem has workaround using existing options
-- Whether the problem can be addressed with a plugin instead
+- is really worth addressing
+- can be fixed with a smarter default
+- has workaround using existing options
+- can be addressed with a plugin instead
 
-## Docs translation contribution
+## Docs Translation Contribution
 
 If you would like to start a translation in your language, you are welcome to contribute! Please join [the #translations channel in Vite Land](https://chat.vitejs.dev) to discuss and coordinate with others.
 
-The english docs are embedded in the main Vite repo, to allow contributors to work on docs, tests and implementation in the same PR. Translations are done by forking the main repo.
+The English docs are embedded in the main Vite repo to enable contributors to work on docs, tests, and implementation in the same PR. Translations are done by forking the main repo.
 
-### How to start a translation repo
+### How To Start a Translation Repo
 
-1. In order to get all doc files, you first need to clone this repo in your personal account.
-2. Keep all the files in `docs/` and remove everything else.
+1. Fork this repo to your personal account.
+2. Keep all the files in `docs/`, and remove everything else.
 
-   - You should setup your translation site based on all the files in `docs/` folder as a VitePress project.
-     (that said, `package.json` is need).
+   - Setup your translation site based on the files in the `docs/` folder as a VitePress project
+     (that said, `package.json` is needed).
 
-   - Refresh git history by removing `.git` and then `git init`
+   - Refresh git history by removing `.git`, and then run `git init`.
 
 3. Translate the docs.
 
-   - During this stage, you may be translating documents and synchronizing updates at the same time, but don't worry about that, it's very common in translation contribution.
```

Comment:
I'm not entirely sure what this line is trying to say. Is it saying that rebasing is commonly needed when working on translations? If so, I think the line could be removed because that seems common in any work.